config.json

```diff
@@ -47,11 +47,15 @@
                 "file_actions": true,
                 "aws_actions": true,
                 "preview_data": true,
-                "glue_crawler": true
+                "glue_crawler": true,
+                "show_stack_logs": "admin-only"
             }
         },
         "s3_datasets_shares": {
-            "active": true
+            "active": true,
+            "features": {
+                "show_stack_logs": "admin-only"
+            }
```

Comment:
From the user experience  show_stack_logs is very similar to show_stack_logs, but implementation-wise they are pretty different. share-logs is a module specific feature that should be defined completely inside the shares module. 
- the config.json parameter should reflect the types of logs we are limiting the visibility for `show_stack_logs` vs `show_share_logs`
- `show_stack_logs` will be implemented in the Stacks core module and `show_share_logs` in the Shares module

Reply:
- Since the logs seen are share generic logs the config.json parameter should not be in `s3_datasets_shares` but in `shares_base`

Reply:
Hi @dlpzx , totally agreed on the naming part. I will make the changes. 
I can put in in another config json - `shares_base`